### PR TITLE
Add how to create a fungible token

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: https://github.com/onflow/docs/issues/396

### CODE REVIEW

File: docs/guides/fungible-token.md [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+    init() {
+        self.totalSupply = 1000.0
+        self.account.save(<- create Minter(), to: /storage/Minter)
+    }
+}
+```
+
+## Deploying the Contract
+
+In order to use the contract, we need to deploy it to the network we want to use it on. In our case we are going to deploy it to emulator while developing.
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:324]:
```suggestion
In order to use the contract, we need to deploy it to the network we want to use it on. In this case we are going to deploy it to emulator while developing. Alternatively we could configure `testnet` or `mainnet`.
```

File: docs/guides/fungible-token.md [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+}
+```
+
+To run this using the CLI, enter this in your terminal:
+
+```
+flow scripts execute cadence/scripts/TotalSupply.cdc
+```
+
+In the terminal where you started the emulator, you should see 1,000.
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:381]:
```suggestion
In the terminal where you started the emulator, you should see `1000.0`
```

File: docs/guides/fungible-token.md [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+	  acct.link<&FooToken.Vault{FungibleToken.Balance, FungibleToken.Receiver}>(/public/Vault, target: /storage/Vault)
+	}
+
+	execute {
+		log("Created")
+	}
+}
+```
+
+Let's next create a new emulator account using the CLI. We'll use this account to create a new vault and mint tokens into it. Run:
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:414]:
```suggestion
Next let's create a new emulator account using the CLI. We'll use this account to create a new vault and mint tokens into it. Run:
```

File: docs/guides/fungible-token.md [hunk 4]
```diff
@@ -0,0 +1,590 @@
...
+flow transactions send ./cadence/transactions/Minter.cdc 123 --signer emulator-account --network emulator
+```
+
+Let's go ahead and read the vault again. Remember to replace `123` with the correct address.
+
+```
+flow scripts execute cadence/scripts/ReadVaultBalance.cdc 123
+```
+
+It should now say 20 are in the vault.
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:515]:
```suggestion
It should now say 20 tokens are in the vault.
```

File: docs/guides/fungible-token.md [hunk 5]
```diff
@@ -0,0 +1,590 @@
...
+
+Make sure to select Emulator as the network.
+
+Don't forget the new account will need a vault added, so let's run the following transaction to add one:
+
+```
+flow transactions send ./cadence/transactions/CreateVault.cdc --signer test-acct-2 --network emulator
+```
+
+Now, let's send one token from our earlier account to the new account. Remember to replace `123` with account address of `test-acct-2`.
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:571]:
```suggestion
Now, let's send 1 token from our earlier account to the new account. Remember to replace `123` with account address of `test-acct-2`.
```

File: docs/guides/fungible-token.md [hunk 6]
```diff
@@ -0,0 +1,590 @@
...
+flow transactions send ./cadence/transactions/Transfer.cdc 123 "1.0" --signer test-acct --network emulator
+```
+
+After that, read the balance of `test-acct-2` (replace the address `123`).
+
+```
+flow scripts execute cadence/scripts/ReadVaultBalance.cdc 123
+```
+
+You should now see one token in `test-acct-2` account!
```
Comment (gregsantos) [inline docs/guides/fungible-token.md:583]:
```suggestion
You should now see 1 token in `test-acct-2` account!
```

**Review by gregsantos — APPROVED** [review]
couple nits but looking great!
Can fix and merge imo

File: docs/guides/fungible-token.md [hunk 7]
```diff
@@ -0,0 +1,590 @@
...
+
+This article will guide you through the steps to create, deploy, mint, and transfer fungible tokens on Flow.
+
+## What are Fungible Tokens?
+
+Fungible tokens are digital assets that are interchangeable and indistinguishable with other tokens of the same type. This means that each token is identical in specification to every other token in circulation. Think of them like traditional money; every dollar bill has the same value as every other dollar bill. Fungible tokens play a crucial role in web3 ecosystems, serving as both a means of payment and an incentive for network participation. They can take on various roles including currencies, structured financial instruments, shares of index funds, and even voting rights in decentralized autonomous organizations.
+
+## Vaults on Flow
+
+On the Flow blockchain, fungible tokens are stored in structures called vaults. Think of a vault as a digital piggy bank. When you transfer tokens from one vault to another:
```
Comment (joshuahannan) [inline docs/guides/fungible-token.md:16]: I would recommend linking to the cadence tutorial or something that goes into a little more detail about how resources are used to represent vaults on Flow for tokens

Comment (gregsantos) [reply docs/guides/fungible-token.md:16]: Feel free to add a PR for this update @joshuahannan. Would be much appreciated!

**Pull request merged**